plugins/poll/lib/poll.rb

```diff
@@ -315,12 +318,12 @@ def self.validate_votes!(poll, options)
     num_of_options = options.length
 
     if poll.multiple?
-      if num_of_options < poll.min
+      if poll.min && (num_of_options < poll.min)
```

Comment:
Just checking, but are `poll.min` and `poll.max` nullable, or is 0 a valid option for them? Because if 0 is a valid option then these may not work correctly.

Reply:
0 is not a valid option for a poll because it just mean that the user is not submitting any votes.